src/v/pandaproxy/handlers.cc

```diff
@@ -61,11 +61,23 @@ ppj::serialization_format parse_serialization_format(std::string_view accept) {
       results, accept, boost::is_any_of(",; "), boost::token_compress_on);
 
     if (std::any_of(results.begin(), results.end(), [](std::string_view v) {
-            return v == "application/vnd.kafka.binary.v2+json";
+            return v == name(ppj::serialization_format::binary_v2);
         })) {
         return ppj::serialization_format::binary_v2;
     }
 
+    if (std::any_of(results.begin(), results.end(), [](std::string_view v) {
+            return v == name(ppj::serialization_format::json_v2);
```

Comment:
why not `static constexpr std::string_view ...foo_bar = name(binary_v2)` and used in all of the places.